audformat/core/scheme.py

```diff
@@ -156,8 +200,9 @@ def draw(
                 seq = string.ascii_letters + string.digits
                 x = [''.join([random.choice(seq) for _ in range(str_len)])
                      for _ in range(n)]
-        elif type(self.labels) in (list, dict):
-            x = [random.choice(list(self.labels)) for _ in range(n)]
+        else:
+            labels = self._labels_to_list(self.labels)
+            x = [random.choice(labels) for _ in range(n)]
```

Comment:
BTW, if a misc table scheme is not assigned to a database `draw()` will return an empty list as `self._labels_to_list(self.labels)` returns `[]` as long as the scheme is not assigned to a database.